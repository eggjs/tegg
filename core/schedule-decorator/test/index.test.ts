import { expect, it } from 'vitest';
import * as types from '../src/index.js';

it('should export stable', async () => {
  expect(types).toMatchSnapshot();
});
