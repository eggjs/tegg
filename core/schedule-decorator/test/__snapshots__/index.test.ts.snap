// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should export stable 1`] = `
{
  "IS_SCHEDULE": Symbol(EggPrototype#isSchedule),
  "SCHEDULE_METADATA": Symbol(EggPrototype#schedule#metadata),
  "SCHEDULE_OPTIONS": Symbol(EggPrototype#schedule#options),
  "SCHEDULE_PARAMS": Symbol(EggPrototype#schedule#params),
  "Schedule": [Function],
  "ScheduleInfoUtil": [Function],
  "ScheduleMetaBuilder": [Function],
  "ScheduleMetadata": [Function],
  "ScheduleMetadataUtil": [Function],
  "ScheduleType": {
    "ALL": "all",
    "WORKER": "worker",
  },
}
`;
